Basic-Car-Maintenance/Shared/Dashboard/Views/AddMaintenanceView.swift

```diff
@@ -98,10 +98,9 @@ struct AddMaintenanceView: View {
 }
 
 #Preview {
-    AddMaintenanceView(vehicles: sampleVehicles) { _ in }
+    @Previewable @State var sampleVehicle = [
```

Comment:
I agree to move this into the `Preview` as opposed to `sampleVehicles` being outside the Preview, but does this need to be  a state property? when really it's only displaying data, as opposed to changing anything